data/groups.yaml

```diff
@@ -176,16 +206,35 @@ mozjs60:
     - mozjs60
 gnuradio:
     name: GNU Radio
+    exception: https://pagure.io/fesco/issue/2272
```

Comment:
Closed, all ported. Give @yarda a medal.

Reply:
Wow! What great news!

This comment appeared to me after I pushed a commit to fixup the group. I'll amend it.

Reply:
I'll remove the group when it disappears in the compose.